tests/testthat/test-gist.R

```diff
@@ -0,0 +1,7 @@
+context("gists")
+
+test_that("animint2gist() returns an object of class 'gist'", {
+  g <- animint2gist(list(p = qplot(1:10)), screenshot = FALSE)
+  expect_is(g, "gist")
+  gistr::delete(g)
```

Comment:
so travis posts a gist and then deletes it with every check now?

If so, AWESOME!